demos/image-classification/index.js

```diff
@@ -16,14 +13,15 @@ import {
   updateQueryStringParameter,
   getMedian,
   getAverage,
+  getMinimum,
   asyncErrorHandling,
-  getMode,
+  getMode
```

Comment:
(minor) Isn't it usually better to retain trailing commas in import lists to reduce the diff noise when adding/removing/reordering parameters?

Reply:
Correct, thanks @fdwr for pointing this out. :)